src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/jpa/Report.kt

```diff
@@ -232,6 +232,7 @@ class Report(
       lastName = lastName,
       createdAt = createdAt,
       text = text,
+      sequence = this.descriptionAddendums.size,
```

Comment:
This is the bit I'm not sure about

Reply:
in other "related objects" the entity's add method requires a sequence to be provided and an example of that is

```kotlin
val sequence = if (report.correctionRequests.isEmpty()) 0 else report.correctionRequests.last().sequence + 1
```
…the logic being that should a thing get deleted the size would be smaller than the last used sequence.

but since there's no deletion of description addenda, your route looks fine to me (for now at least!)